src/graphics/Window.cpp

```diff
@@ -6,51 +6,33 @@
 
 namespace Bald::Graphics {
 
-    Window::Window(const char* title, int width, int height, bool closed)
-            : m_Title(title), m_Width(width), m_Height(height), m_Closed(closed) {
-        Init();
-    }
-
-    Window::~Window() {
-        glfwTerminate();
-    }
-
-
-    void Window::Update() const {
-        if(!m_Closed){
+    constexpr void Window::Update(){
+        if (!m_Closed) {
             glfwPollEvents();
+            glfwGetFramebufferSize(m_Window, &m_Width, &m_Height);
+            glViewport(0, 0, m_Width, m_Height);
             glfwSwapBuffers(m_Window);
         }
-
     }
 
-
     bool Window::Init() noexcept {
 
```

Comment:
Is it OK to re-init same window one more time?
Maybe there's if(!m_Closed) check needed?

Reply:
maybe just make it private

Reply:
Method **Init()** is private indeed